spec/rubocop/cop/rspec/unspecified_exception_spec.rb

```diff
@@ -243,15 +243,10 @@ def raise_error
       RUBY
     end
 
-    it 'allows a subject function to be named raise_error' do
```

Comment:
Let’s keep this test case.

Reply:
I'll put it back. Just so you know, I replaced it because it would be very hard to break this spec without also breaking the spec above it. You'd have to specifically add special handling to do so for only one of the method names, which are not referenced outside `RESTRICT_ON_SEND`

Reply:
Ah, I see. Anyway, let’s keep both anyway.